examples/glasplaat.json

```diff
@@ -0,0 +1,171 @@
+      "Vlakbeelden Midden Rijk"
+    ]
+  },
+  "metadata": [
+    {
+      "label": {
+        "en": [
+          "Type"
+        ]
+      },
+      "value": {
+        "en": [
+          "Photo"
+        ]
+      }
+    },
+    {
+      "label": {
+        "en": [
+          "Date"
+        ]
+      },
+      "value": {
+        "none": [
+          "1900/1960 <dcterms:created^edtf>"
+        ]
+      }
+    },
+    {
+      "label": {
+        "en": [
+          "pid"
+        ]
+      },
+      "value": {
+        "none": [
+          "5d8nc8tp93"
+        ]
+      }
+    },
+    {
+      "label": {
+        "en": [
+          "Identifiers"
+        ]
+      },
+      "value": {
+        "none": [
+          "BIB.GLAS.016209"
+        ]
+      }
+    }
+  ],
+  "thumbnail": [
+    {
+      "id": "https://images.meemoo.be/iiif/public/OR-0z70w1b/f6/f64922b8a6314067a1a928119060f51659de2ef8245246e3b8676111c9adc34b9bf8c4f666c14a41936dbeb98669fe4d/full/480,/0/default.jpg",
+      "type": "Image",
+      "format": "image/jpeg",
+      "service": [
+        {
+          "id": "https://images.meemoo.be/iiif/public/OR-0z70w1b/f6/f64922b8a6314067a1a928119060f51659de2ef8245246e3b8676111c9adc34b9bf8c4f666c14a41936dbeb98669fe4d",
+          "protocol": "http://iiif.io/api/image",
+          "profile": "level1"
+        }
+      ]
+    }
+  ],
+  "requiredStatement": {
+    "label": {
+      "en": [
+        "Attribution"
+      ],
+      "nl": [
+        "Attributie"
+      ]
+    },
+    "value": {
+      "none": [
+        "Aangeboden door UGent"
+      ]
+    }
+  },
+  "rights": "<schema:license> Er is geen CC of Public Domain license toegekend",
+  "items": [
+    {
+      "id": "https://iiif.meemoo.be/presentation/3/5d8nc8tp93/canvas/1",
+      "type": "Canvas",
+      "label": {
+        "none": [
+          "<schema:name>"
+        ]
+      },
+      "thumbnail": [
+        {
+          "id": "https://images.meemoo.be/iiif/public/OR-0z70w1b/f6/f64922b8a6314067a1a928119060f51659de2ef8245246e3b8676111c9adc34b9bf8c4f666c14a41936dbeb98669fe4d/full/480,/0/default.jpg",
+          "type": "Image",
+          "format": "image/jpeg",
+          "service": [
+            {
+              "id": "https://images.meemoo.be/iiif/public/OR-0z70w1b/f6/f64922b8a6314067a1a928119060f51659de2ef8245246e3b8676111c9adc34b9bf8c4f666c14a41936dbeb98669fe4d",
+              "protocol": "http://iiif.io/api/image",
+              "profile": "level1"
+            }
+          ]
+        }
+      ],
+      "items": [
+        {
+          "id": "https://iiif.meemoo.be/presentation/3/5d8nc8tp93/canvas/1/page/1",
+          "type": "AnnotationPage",
+          "items": [
+            {
+              "id": "https://iiif.meemoo.be/presentation/3/5d8nc8tp93/canvas/1/page/1/annotation/1",
+              "type": "Annotation",
+              "motivation": "painting",
+              "target": "https://iiif.meemoo.be/presentation/3/5d8nc8tp93/canvas/1",
+              "body": {
```

Comment:
de content heeft geen `height`en `width`. dat is niet verplicht, maar wel aanbevolen en wordt meestal ook gedaan.

Reply:
we hebben deze data niet in de KG, daarom worden ze vooralsnog niet gebruijt/getoond hoewel we dit liever wel zouden kunnen